### ISSUE TITLE
clean up old npm release tags

### ISSUE THREAD

**Issue by holic** [latticexyz/mud#2984]
there's a bunch in there that are no longer relevant and make it hard to find real/maintained tags
https://www.npmjs.com/package/@latticexyz/cli?activeTab=versions

<img width="1255" alt="image" src="https://github.com/user-attachments/assets/654ac2e9-653c-46a7-ad4c-18a671b0a02a">

**Comment by holic** [comment]
there appears to be an issue with `npm dist-tag rm` where the target tag will disappear until you run the command again with a different tag, then the previously deleted one reappears

I've emailed npm support about this and wondering if this means that tags can no longer be deleted, only deprecated?

**Comment by holic** [comment]
blocked on https://github.com/npm/cli/issues/7705

**Comment by holic** [comment]
the issue at npm seems to be resolved and was able to clean these up with our "Remove npm tag" action
